CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Cromwell Change Log
 
+## 30 Release Notes
+
+### Other changes
+
+* **Database**  
+If desired, the metadata may be optionally stored in a separate SQL database than the of the operations data. See the
+[README](https://github.com/broadinstitute/cromwell#database) or the `database` section in
+[cromwell.examples.conf](https://github.com/broadinstitute/cromwell/blob/develop/cromwell.examples.conf) for more
+information. Cromwell will not move any existing data automatically. When switching connection information for an
```

Comment:
Something which happens frequently is that a user will see a note like this in the release notes and then ask how they'd know if they'd want to do this.

Because I think our changelog is already way too verbose I don't want to see such an explanation here, but it'd be good to find a way to nip those Qs in the bud.